### PR TITLE
[FCE-481] Fix local e2e example tests

### PR DESCRIPTION
**Pull request by MiloszFilimowski**
## Description

- Changed appium method to swipeGesture for closing select audio source
- Enabled all tests for Android 
- Disabled some tests for iOS and added a TODO that should be fixed here: https://linear.app/swmansion/issue/FCE-504/fix-e2e-test-on-ios-related-to-showing-camera-view-when-camera-is

## Motivation and Context

- Some tests were disabled on Github and failed locally. This was because it was not possible to close the audio source selection input.

## How has this been tested?

- Run Android tests both locally and on Github Actions
- Run remaining iOS tests locally

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the code style of this project.

### CODE REVIEW

File: examples/fishjam-chat/components/SoundOutputDevicesBottomSheet.tsx [hunk 1]
```diff
@@ -3,6 +3,9 @@ import React, { useCallback, useState } from 'react';
 import { StyleSheet } from 'react-native';
 
 import SoundOutputDevicesSection from './SoundOutputDevicesSection';
+import { soundOutputDevicesLabels } from '../types/ComponentLabels';
+
+const { OUTPUT_DEVICE_BOTTOM_SHEET } = soundOutputDevicesLabels;
```
Comment (karkakol) [inline examples/fishjam-chat/components/SoundOutputDevicesBottomSheet.tsx:8]:
```suggestion
const { OUTPUT_DEVICES_BOTTOM_SHEET } = soundOutputDevicesLabels;
```

File: examples/fishjam-chat/types/ComponentLabels.ts [hunk 2]
```diff
@@ -33,6 +33,7 @@ const roomScreenLabels = {
 const soundOutputDevicesLabels = {
   TITLE_TEXT: 'title_text',
   OUTPUT_DEVICE_BUTTON: 'output_device_button',
+  OUTPUT_DEVICE_BOTTOM_SHEET: 'output_device_bottom_sheet',
```
Comment (karkakol) [inline examples/fishjam-chat/types/ComponentLabels.ts:36]:
```suggestion
  OUTPUT_DEVICE_BOTTOM_SHEET: 'output_devices_bottom_sheet',
```

File: examples/webdriverio-test/utils.ts [hunk 3]
```diff
@@ -64,6 +64,18 @@ const tapApp = async (driver: WebdriverIO.Browser) => {
...
+
+  await driver.executeScript('mobile: swipeGesture', [
+    {
+      elementId: element.elementId,
+      direction: 'down',
+      percent: 1.0,
+    },
+  ]);
+};
+
```
Comment (karkakol) [inline examples/webdriverio-test/utils.ts:78]: 🥹 
👍

**Review by karkakol — APPROVED** [review]

File: examples/webdriverio-test/test/specs/test.e2e.ts [hunk 4]
```diff
@@ -46,6 +47,11 @@ const {
 
 const { TITLE_TEXT, OUTPUT_DEVICE_BUTTON } = soundOutputDevicesLabels;
 
+const { OUTPUT_DEVICES_BOTTOM_SHEET } = soundOutputDevicesLabels;
+
+// TODO: Remove after fixing: https://linear.app/swmansion/issue/FCE-504/fix-e2e-test-on-ios-related-to-showing-camera-view-when-camera-is
```
Comment (mironiasty) [inline examples/webdriverio-test/test/specs/test.e2e.ts:52]: nit: you can just leave ticket number:
```suggestion
// TODO: Remove after fixing: FCE-504
```

**Pull request merged**